docs/docs/Integrations/Google/integrations-google-big-query.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Integrate Google BigQuery with Langflow
+slug: /integrations-google-big-query
+description: "A comprehensive guide on setting up and using the BigQuery component in Langflow to execute SQL queries on Google BigQuery."
```

Comment:
Is the description always in quotes? I don't recall seeing that in previous PRs, but maybe I just never noticed.

Reply:
The description field isn't used elsewhere- it only remained in the Google integrations pages, so I have removed it. 
Per the quotes, you're right, Docusaurus does not have quotation marks around [their example. ](https://docusaurus.io/docs/markdown-features/head-metadata#markdown-page-description)